core/src/main/java/org/sonarsource/sonarlint/core/container/storage/StorageManager.java

```diff
@@ -194,6 +212,11 @@ public Date getLastUpdateDate() {
         public boolean isStale() {
           return stale;
         }
+
+        @Override
+        public Set<String> unusedRepoKeys() {
+          return null;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Return an empty collection instead of null. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1168)